lms/static/sass/base/_buttons.scss

```diff
@@ -0,0 +1,76 @@
+
+@mixin primary-button ($isImportant:null) {
+    color: #FFF $isImportant;
+    font-size: 12px $isImportant;
+    text-transform: uppercase $isImportant;
+    border-radius: 2px $isImportant;
+    padding: 3px 15px $isImportant;
+    font-weight: normal $isImportant;
+    background-color: $mckinsey-primary-default-button $isImportant;
+    height: 40px !important;
+}
+
+@mixin primary-button-hover ($isImportant:null) {
+    color: #FFF $isImportant;
+    font-size: 12px $isImportant;
+    text-transform: uppercase $isImportant;
+    border-radius: 2px $isImportant;
+    padding: 3px 15px $isImportant;
+    font-weight: normal $isImportant;
+    background-color: #0b0886 !important;
+    height: 40px !important;
+
+}
+
+@mixin primary-button-large ($isImportant:null) {
+    @include primary-button($isImportant);
+    height: 50px !important;
+    font-size: 16px $isImportant;
```

Comment:
@prakashpathak02 Please convert you font sizes in rem with the reference I shared via email